packages/@divvi/mobile/src/tokens/selectors.ts

```diff
@@ -558,12 +519,12 @@ const getJumpstartEnabledNetworkIds = () =>
   ) as NetworkId[]
 
 export const jumpstartSendTokensSelector = createSelector(
-  [(state) => sortedTokensWithBalanceSelector(state, getJumpstartEnabledNetworkIds())],
+  sortedTokensWithBalanceSelector,
   (tokensWithBalance) => {
     return tokensWithBalance.filter((token) => {
       // the jumpstart contract currently requires a token address for the
       // depositERC20 method
-      return !!token.address
+      return getJumpstartEnabledNetworkIds().includes(token.networkId) && !!token.address
```

Comment:
maybe worth adding `getJumpstartEnabledNetworkIds` to the `createSelector` inputs for best practise? it's unlikely to make any meaningful difference though since the jumpstart network ids are very unlikely to change 😅

Reply:
couldn't do it with createSelector, ended up using a regular function.